lib/archive/zip/entry.rb

```diff
@@ -932,7 +932,6 @@ def link_target
     # Sets the link target for this entry.  As a side effect, the raw_data
     # attribute is set to +nil+.
     def link_target=(link_target)
-      raw_data = nil
```

Comment:
Sorry I didn't catch this one earlier.  As documented for this method, we need to set the `raw_data` attribute to `nil` here.  It should actually be done as follows:

```ruby
self.raw_data = nil
```

Would you mind making that small edit?